app/controllers/reactions_controller.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+class ReactionsController < ApplicationController
+  skip_before_action :verify_authenticity_token
+  before_action :set_chirp
+
+  def create
+    current_user.reactions.create!(chirp: @chirp, type: params[:reaction_type])
+
+    respond_to do |format|
+      format.html do
+        redirect_back fallback_location: user_path(@user)
+      end
+      format.json do
+        render json: ChirpPresenter.to_hash(@chirp.reload, current_user)
```

Comment:
Looking at the code again today, I do wonder if the response for reactions#create makes sense? I copied pasted it from elsewhere and it returns the whole `ChirpPresenter.to_hash` 🤔 might be better to just return the `reactions` key from that presenter response?